test/basedriver/helpers-specs.js

```diff
@@ -1,7 +1,8 @@
-import { isPackageOrBundle, duplicateKeys, parseCapsArray } from '../../lib/basedriver/helpers';
+import { zip, fs, tempDir } from 'appium-support';
+import { configureApp, isPackageOrBundle, duplicateKeys, parseCapsArray } from '../../lib/basedriver/helpers';
```

Comment:
this line seems to be too long

Reply:
imo we shouldn't concern ourselves with formatting changes (unless it fails lint) if the formatting is not automated (e.g., precommit hook); there are bigger fish to fry

(the formatting _is_ automated in the `appium/appium` monorepo)